src/App.js

```diff
@@ -154,7 +154,6 @@ class App extends Component {
     //console.log(source);
     this.setState({
       selectedSource: source,
-      showLoader: false
```

Comment:
I don't think that you need to change the loader state when you select a new source, since you don't fetch any new articles (yet)